build.psm1

```diff
@@ -992,7 +992,7 @@ function Start-PSPester {
         [switch]$IncludeFailingTest
     )
 
-    if (-not (Get-Module -ListAvailable -Name $Pester -ErrorAction SilentlyContinue))
+    if (-not (Get-Module -ListAvailable -Name $Pester -ErrorAction SilentlyContinue | Where { $_.Version -ge "4.2" } ))
```

Comment:
Maybe we should add a specific error that the expected version is not found.

Reply:
Special warning for too low version added.